sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveComparisonTest.scala

```diff
@@ -480,7 +480,7 @@ abstract class HiveComparisonTest
                 val executions = queryList.map(new TestHive.QueryExecution(_))
                 executions.foreach(_.toRdd)
                 val tablesGenerated = queryList.zip(executions).flatMap {
-                  case (q, e) => e.sparkPlan.collect {
+                  case (q, e) => e.executedPlan.collect {
```

Comment:
Yea. But, at here, we are only collecting `InsertIntoHiveTable` nodes. Why using executedPlan at here solve the problem?